dlgr/griduniverse/static/scripts/demo.js

```diff
@@ -1076,6 +996,54 @@ function updateDonationStatus(donation_is_active) {
   settings.donation_active = donation_is_active;
 }
 
+
+function renderTransition(transition) {
+  if (! transition) {
+    return "";
+  }
+  const states = [
+    transition.transition.actor_start,
+    transition.transition.actor_end,
+    transition.transition.target_start,
+    transition.transition.target_end
+  ];
+
+  const [aStartItem, aEndItem, tStartItem, tEndItem] = states.map(
+    (state) => settings.item_config[state]
+  );
+
+  return `✋${aStartItem.name} + ${tStartItem.name} = ✋${aEndItem.name} + ${tEndItem.name}`;
+}
+/**
+ * If the current player is sharing a grid position with an interactive
+ * item, we show information about it on the page.
+ *
+ * @param {Player} egoPlayer the current Player
+ * @param {itemlib.GridItems} gridItems  the collection of all Items on the grid
+ */
+function updateItemInfoWindow(egoPlayer, gridItems) {
+  const inspectedItem = gridItems.atPosition(egoPlayer.position),
+        transition = egoPlayer.getTransition(),
+        $square = $("#location-contents-item"),
+        $transition = $("#transition-details");
+
+  if (_.isUndefined(inspectedItem)) {
+    $square.empty();
+  } else {
+    $square.html(inspectedItem.name);
+  }
+
+  if (_.isUndefined(transition)) {
```

Comment:
`player.getTransition()` should return `null` not `undefined`. I think it would be more semantically appropriate if `gridItems.atPosition(...)` returned `null` as well.